pages/howto/matlab.md

```diff
@@ -448,35 +448,35 @@ Download the [CCR Matlab Integration Scripts](https://g-ac407a.8c185.08cc.data.g
 
 ans =
 
-    '/home/testuser/Documents/MATLAB'
+    '/home/[username]/Documents/MATLAB'
 
 >> 
 
 ```
-
+**NOTE:** This is on your local computer so the directory path will most likely be different.
 
 Configure your local MATLAB to run parallel jobs on the cluster by using the `configCluster` cluster command.  This only needs to be run once per version of MATLAB.
 
 
 ```
 >> configCluster
-Username on UB-HPC (e.g. jdoe): >>testuser
+Username on UB-HPC (e.g. jdoe): >>[CCRusername]
 Complete.  Default cluster profile set to "ub-hpc R2023b".
 >> 
 
 ```
-You will need to configure your user infomation into the Profile including the location to your private SSH key:
+You will need to configure your user infomation into the Profile including the location to your private SSH key stored on your computer:
 
 ```
 >> c = parcluster;
->> c.AdditionalProperties.EmailAddress = 'testuser@buffalo.edu'
->> c.AdditionalProperties.IdentityFile = '/home/testuser/.ssh/private.key'
+>> c.AdditionalProperties.EmailAddress = 'UBITusername@buffalo.edu'
```

Comment:
Maybe:
[UBITusername]@buffalo.edu

Users may be more likely to confuse UBITusername and CCRusername if both are bracketed, but in this case it is a variable we want users to substitute out.